pytomorrowio/pytomorrowio.py

```diff
@@ -385,29 +382,26 @@ async def realtime_and_all_forecasts(
 
         forecasts: Dict[str, List[Dict[str, Any]]] = {}
         if all_forecasts_fields:
-            forecasts = await TomorrowioV4.all_forecasts(
-                self,
+            forecasts = await self.all_forecasts(
```

Comment:
this is needed for the synchronous functions to work, please revert. It's a bit of a hack but the sync version makes it easy to run quick tests

Reply:
what happens is the Sync class is just calling the parent functions but with a decorator that calls the function with `asyncio.run`. If one function calls another function, and that function is defined in the sync class, then `asyncio.run` gets wrapped in another `asyncio.run` because the sync class will call the sync function instead of the async function in the base class. So here we are forcing the function to call the `TomorrowioV4` version of `forecast` etc. instead of the `TomorrowioV4Sync` one

Reply:
btw if you have another suggestion on how to create a sync version of the async class without duplicating code, I am all ears. I found this solution a while ago and it worked for my basic needs so I didn't dig much further than this.

Reply:
Ah, that makes sense (I forgot that it was using inheritance). I'll what I can find... tomorrow. :)